src/syft/ast/globals.py

```diff
@@ -18,6 +18,22 @@ class Globals(Module):
     def __init__(self) -> None:
         super().__init__("globals")
 
+    def __call__(
```

Comment:
Q: Did we had a callable class for this? (what is the usecase?)

Reply:
So the parent is callable, but the signature needs to be augmented so that the new obj_type param flows through. This is a fix for the FQN lookup for stuff like torch.nn.modules.conv.Conv2d vs torch.nn.Conv2d